api/tests/routes/backoffice/individual_bookings_test.py

```diff
@@ -297,6 +297,39 @@ def test_list_bookings_by_offer_id(self, authenticated_client, bookings):
 
         assert html_parser.extract_pagination_info(response.data) == (1, 1, len(rows))
 
+    @pytest.mark.parametrize(
+        "cancellation_reason",
+        [
+            (bookings_models.BookingCancellationReasons.BACKOFFICE),
+            (bookings_models.BookingCancellationReasons.OFFERER),
+            (bookings_models.BookingCancellationReasons.FRAUD),
+        ],
+    )
+    def test_list_cancelled_booking_informations(self, authenticated_client, legit_user, cancellation_reason):
```

Comment:
```suggestion
    def test_list_cancelled_booking_information(self, authenticated_client, legit_user, cancellation_reason):
```
Information est un indénombrable, en anglais

Reply:
Par ailleurs, ton test n'est pas très bien placé, entre des tests de filtre; tu peux le mettre en fin de classe.